Sources/ArcGISToolkit/Components/Authentication/Authenticator.swift

```diff
@@ -37,6 +37,12 @@ public final class Authenticator: ObservableObject {
         self.oAuthUserConfigurations = oAuthUserConfigurations
     }
     
+    /// Sets the ArcGIS and Network challenge handlers to be this authenticator.
+    public func setupAuthenticationChallengeHandlers() {
```

Comment:
I'm not opposed to this idea but I'm not sure about this name.  I think something more like "become the default challenge handler" or something like that would be ideal.  Phil R might have some ideas for the name of this method.

Reply:
I think it would make more sense as a method on the authentication manager. Something like this:

```swift
public extension AuthenticationManager {
    func setAuthenticator(_ authenticator: Authenticator {
        arcGISAuthenticationChallengeHandler = authenticator
        networkAuthenticationChallengeHandler = authenticator
    }
}
```

Or `useAuthenticator(_:)` or `handleChallenges(using:)` or …

Reply:
As an instance method on `Authenticator`, it isn't clear (without looking at the documentation) that it modifies shared mutable state. Putting it on the authentication manager makes that explicit.

Reply:
Ok, then should it be done for `setupPersistentCredentialStorage` at line 53 too?

Reply:
Thanks @philium. Moved 3 functions in the `AuthenticationManager` extension.